Documentation/PropertyWrapper/Parameter.md

```diff
@@ -295,7 +295,7 @@ struct Bird: Codable, Identifiable {
 }
 
 struct ExampleComponent: Component {
-    @PathParameter var birdID: Bird.ID
+    @Parameter(.path) var birdID: Bird.ID
 
     // ...
 }
```

Comment:
I'm unsure what the REST/OpenAPI exporter would do with that path parameter in this example?
The `ExampleComponent` is defined under the path `/api/birds` and the `@Parameter(.path)` is not explicitly included in the PathComponents.
Should the REST/OpenAPI exporter assume a path parameter on the top level then? My assumption was that a `@Parameter` which should be a path component must be explicitly declared as in the example below, or have I mixed up something?

Reply:
Just noticed the short disclaimer below the code 😅
Are we sure that we want to allow this implicit definition of path parameters? I would argue against it as one loses the overview, as the "structure" of the Webservice could now be hidden inside of Components.

Reply:
That's a good point. The proposal would suggest that the `@Parameter(.path)` parameters which were not explicitly passed into a `Component` are added at the end of the path for RESTful APIs. While this is not explicitly visible in the DSL, I would suggest it is fine that this is sometime defined in the `Component` itself.